packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -248,6 +249,144 @@ describe("Integration tests", () => {
         );
       });
     });
+    describe("updateVerifiedAttributeExtensionDate", async () => {
+      const tenant: Tenant = {
+        ...mockTenant,
+        attributes: [
+          {
+            ...mockVerifiedTenantAttribute,
+            verifiedBy: [
+              {
+                ...mockVerifiedBy,
+                extensionDate: currentDate,
+                expirationDate,
+              },
+            ],
+          },
+        ],
+        name: "A Tenant",
+      };
+      const attributeId = tenant.attributes.map((a) => a.id)[0];
+      const verifierId = mockVerifiedBy.id;
+      it("Should update the extensionDate", async () => {
+        const extensionDate = new Date(
+          currentDate.getTime() +
+            (expirationDate.getTime() -
+              mockVerifiedBy.verificationDate.getTime())
+        );
+
+        await addOneTenant(tenant, postgresDB, tenants);
+        await tenantService.updateVerifiedAttributeExtensionDate(
+          tenant.id,
+          attributeId,
+          verifierId
+        );
+        const writtenEvent = await readLastEventByStreamId(
+          tenant.id,
+          postgresDB
+        );
+        expect(writtenEvent.stream_id).toBe(tenant.id);
+        expect(writtenEvent.version).toBe("1");
+        expect(writtenEvent.type).toBe("TenantUpdated");
+        const writtenPayload = decodeProtobufPayload({
+          messageType: TenantUpdatedV1,
+          payload: writtenEvent.data,
+        });
+
+        const updatedTenant: Tenant = {
+          ...tenant,
+          attributes: [
+            {
+              ...mockVerifiedTenantAttribute,
+              verifiedBy: [
+                {
+                  ...mockVerifiedBy,
```

Comment:
It would be better if you get these from `tenant` instead getting them from the mocks, because `updatedTenant` is based on `tenant`